server/grpc_cas.go

```diff
@@ -90,8 +90,8 @@ func (s *grpcServer) BatchUpdateBlobs(ctx context.Context,
 // Return the data for a blob, or an error.  If the blob was not
 // found, the returned error is errBlobNotFound. Only use this
 // function when it's OK to buffer the entire blob in memory.
-func (s *grpcServer) getBlobData(hash string, size int64) ([]byte, error) {
-	rdr, sizeBytes, err := s.cache.Get(cache.CAS, hash)
+func (s *grpcServer) getBlobData(instanceName string, hash string, size int64) ([]byte, error) {
```

Comment:
The instanceName arg seems to be useless here too, maybe just skip it and use the empty string?